src/libUtils/SafeMath.tpp

```diff
@@ -271,10 +271,6 @@ bool SafeMath<T>::div_signint(const T& a, const T& b, T& result) {
 
   T c = a / b;
```

Comment:
Why didn't check if b is zero?

Reply:
The check for that is in `SafeMath<T>::div`, which calls this function

Reply:
ok